src/platform/graphics/webgl/webgl-texture.js

```diff
@@ -451,6 +462,41 @@ class WebglTexture {
                                   this._glPixelType,
                                   mipObject);
                 }
+            } else if (texture._array && typeof mipObject === "object") {
+                if (texture._arrayLength === mipObject.length) {
+                    if (texture._compressed) {
+                        for (let index = 0; index < texture._arrayLength; index++) {
+                            gl.compressedTexSubImage3D(
+                                gl.TEXTURE_2D_ARRAY,
+                                mipLevel,
+                                0,
+                                0,
+                                index,
+                                Math.max(texture._width * resMult, 1),
```

Comment:
resMult is float .. and so we should use Math.floor here, for the cases it's not a power of 2

Reply:
Updated as requested.